dotcom-rendering/src/layouts/CommentLayout.tsx

```diff
@@ -779,7 +779,7 @@ export const CommentLayout = (props: WebProps | AppsProps) => {
 							>
 								<RightColumn>
 									<MostViewedRightWithAd
-										display={format.display}
+										format={format}
```

Comment:
do we use anything other than display from format here?

Reply:
I've there's now a check for the design being video to set the label to be dark, but I suppose I could do that via the palette!

Reply:
We do, to decide if it needs to have a dark colour scheme!